test/fixtures/index.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
   inputPngBooleanNoAlpha: getPath('bandbool.png'),
   inputPngTestJoinChannel: getPath('testJoinChannel.png'),
   inputPngTruncated: getPath('truncated.png'), // gm convert 2569067123_aca715a2ee_o.jpg -resize 320x240 saw.png ; head -c 10000 saw.png > truncated.png
+  inputPngEmbed: getPath('embedgravitybird.png'),
```

Comment:
Text on the image mentions you own the copyright for this test image. Under what licence is sharp granted use to it?

Reply:
I am releasing it to sharp under a Creative Commons Attribution 4.0 International License. Do I need to specify this somewhere in the project files?

Reply:
Great, thank you, please can you add "CC BY 4.0" as an inline comment so it's in the git history.